### ISSUE TITLE
Object documentation: PoliticalDonation

### ISSUE THREAD

**Issue by lottspot** [Lever-age/leverage#52]
https://github.com/Lever-age/api/wiki/Interface#politicaldonation

**Comment by lottspot** [comment]
Looks like we're still waiting on a description for `is_fixed_asset`

**Comment by lottspot** [comment]
On closer inspection, it looks like we don't actually need to include this object in the public API documentation, since it is never exposed publicly (thankfully -- we would have a big problem on our hands if it was). It'll be good to have a separate internal documentation though, so we'll keep this documented with the API objects for now until we can split it out into a separate doc. In any case, this particular task shouldn't be kept open.
